src/main/java/bio/terra/service/load/LoadService.java

```diff
@@ -0,0 +1,55 @@
+package bio.terra.service.load;
+
+import bio.terra.service.load.exception.LoadLockFailureException;
+import bio.terra.service.load.flight.LoadMapKeys;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.FlightMap;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+
+@Component
+public class LoadService {
+    private final LoadDao loadDao;
+
+    @Autowired
+    public LoadService(LoadDao loadDao) {
+        this.loadDao = loadDao;
+    }
+
+    public void lockLoad(String loadTag, String flightId) {
+        loadDao.lockLoad(loadTag, flightId);
+    }
+
+    public void unlockLoad(String loadTag, String flightId) {
+        loadDao.unlockLoad(loadTag, flightId);
+    }
+
+    /**
+     * @param inputTag  may be null or blank
+     * @return either valid inputTag or generated date-time tag.
+     */
+    public String computeLoadTag(String inputTag) {
```

Comment:
Worth unit-testing to be sure nothing weird happens when you pass in `null`?

Reply:
It is exercised in functional tests, so I didn't think a unit test was necessary. Easy enough to add.